services/QuillLMS/app/views/pages/diagnostic_tool.erb

```diff
@@ -1,250 +1,89 @@
+    </div>
+  </header>
+  <section class="tool-on-laptop">
+    <img alt="Screenshot of a Quill diagnostic activity full-screen on a laptop" src=<%= "#{ENV['CDN_URL']}/images/tools/diagnostic-on-laptop.svg" %> >
+    <%= render_video_content(@device, 'desktop', "#{diagnostic_animation_url}") %>
+    <%= render_video_content(@device, 'mobile', "#{diagnostic_animation_url}") %>
+  </section>
+  <section class="tool-ctas">
+    <a href=<%= "/activity_sessions/anonymous?activity_id=ffd22b0d-cafe-42fb-bc96-5d5a02a40a55" %> rel="noopener noreferrer" target="_blank" class="quill-button contained extra-large red focus-on-light">Try a sample activity</a>
+    <% if current_user&.teacher? %>
+      <a href='/assign/activity-library?activityClassificationFilters[]=diagnostic' class="quill-button contained extra-large red focus-on-light">Assign</a>
+    <% end %>
+  </section>
+  <%= render partial: "pages/shared/research_blocks", locals: { research_blocks: [
+    { icon: "teacher-whiteboard-red.svg", title: "Student Responses Analyzed", description: "Quill automatically analyzes and grades open-ended responses. It identifies the writing skills students have mastered, as well as the ones they need to practice." },
+    { icon: "student-study-red.svg", title: "Recommended Activities", description: "You can then view a variety of reports to identify individual student and class-wide learning gaps. Quill generates a personalized learning plan for students based on where they need the most help." },
+    { icon: "teacher-whiteboard-red.svg", title: "Evidence Based", description: "The diagnostics evaluate student writing using an evidence based strategy called sentence combining. This strategy explicitly teaches students how to write sophisticated, complex sentences." },
+  ] } %>
+  <div class="targeted-skills-section">
+  <%= render partial: "pages/shared/targeted_skills", locals: {
+    heading: "What's Being Tested",
+    description: "<p>The diagnostics cover fundamental areas of sentence structure, giving you insight into your students' ability to effectively create both basic and advanced sentences.</p>",
+    tags: [
+      "Subject-Verb Agreement",
+      "Capitalization",
+      "Compound Subjects, Objects, Predicates",
+      "Compound Sentences",
+      "Complex Sentences",
+      "Adjectives & Modifying Phrases",
+      "Parallel Structure",
+    ],
+    buttons: [
+      { text: "View Starter Diagnostic", href: "/activity_sessions/anonymous?activity_id=2537", color: "red" },
+      { text: "View Intermediate Diagnostic", href: "/activity_sessions/anonymous?activity_id=2539", color: "red" },
+      { text: "View Advanced Diagnostic", href: "/activity_sessions/anonymous?activity_id=2541", color: "red" }
```

Comment:
We have these ids stored in the `activity.rb` file, I wonder if it's worth passing them up as variables to this erb file to make it a little less brittle in case we change those again in the future?

Reply:
good suggestion! I'll do that